docs/shared_parsers_catalog/neutron_plugin.rst

```diff
@@ -0,0 +1,9 @@
+Neutron Plugin Configuration
+============================
+
+file ``/etc/neutron/plugin.ini``
```

Comment:
This is indeed a new method to write the `.rst` file, however, it will override the `title` and `header` (line 5 & 6) in file `neutron_plugin.py`. Do you really want to hide the content (line 2~6 of file "neutron_plugin.py") in the documentation?